frontend/src/app/shared/components/grids/widgets/add/add.modal.html

```diff
@@ -37,11 +37,11 @@
     </ul>
   </div>
 
-  <div class="spot-action-bar">
+  <div class="spot-action-bar spot-modal--cancel-button">
```

Comment:
Hi @oliverguenther, I think there should be  `hidden-for-mobile` instead of `spot-modal--cancel-button` here for action bar. Other than that, everything is fine.

Reply:
Good catch, thanks! :bow: